src/db.js

```diff
@@ -23,7 +23,7 @@ const setConnect = () => {
 };
 
 class MongoConnetor {
-  connect = async () => {
+  connect = () => {
```

Comment:
This should be a wrapped promise of the `.connect` call, and return the connection object